CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/sna/centrality/PathScoringUtilities.java

```diff
@@ -31,6 +31,10 @@
  * @author cygnus_x-1
  */
 public class PathScoringUtilities {
+    
+    private static final String SCORETYPE_ERROR_STRING1 = "The requested ScoreType, ";
```

Comment:
Perhaps make this "The requested ScoreType, %s, is not supported." and use a string formatter?

Reply:
Can do this but worth noting that sonar will likely flag this as needing a constant anyway

Reply:
Done

Reply:
You could still use a constant for the string.